src/Uno.Extensions.Reactive.UI.Tests/Presentation/Bindings/Collections/Given_BindableCollection_Selection.cs

```diff
@@ -107,4 +181,16 @@ private async Task<(BindableGiven_BindableCollection_Selection_Model, ListView,
 
 		return (vm, lv, lvItems);
 	}
+
+	private async Task<(BindableGiven_BindableCollection_Selection_Model, ComboBox, ICollectionView, MyItem[])> SetupComboBox()
+	{
+		var vm = new BindableGiven_BindableCollection_Selection_Model();
+		var cb = new ComboBox { ItemsSource = vm.Items };
+
+		await UIHelper.Load(cb, CT);
+
+		var items = vm.Items.Value.ToArray();
```

Comment:
The `.Value` doesn't not exists on an `IFeed` ... copilot is just an helper, don't trust it 😊 Anyway you don't need that, you're not using them in your test methods. 

Note: You als have to remove the `ICollectionView` from the return tuple type of your method ;)